iwf.yaml

```diff
@@ -1512,9 +1512,9 @@ components:
           type: string
         stateExecutionId: # only applicable for certain events
           type: string
-        startTimestamp: # only applicable for certain events
+        startTimestampInMs: # only applicable for certain events
```

Comment:
WDYT we also change `E2E` to `EE` or `ETOE`? 
It's not a problem for our actual metric because we can choose different name, but the golang code will look really weird 🤦

Reply:
I'll change it if it really bugs you so much 😂 I think `E2_E` looks better than `ETOE`. `EE` is okay with me. Are you cool with `EE`?

Reply:
yeah EE is fine to me.

Reply:
Done